pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -41,6 +43,10 @@ type (
 		// +optional
 		CIDR *ipnet.IPNet `json:"cidr,omitempty"`
 	}
+	// VpcNAT holds NAT gateway configuration options
+	VpcNAT struct {
```

Comment:
How about `ClusterNAT`? It would be in line with `ClusterSubnets` and `ClusterVPC` :)

Reply:
Cool :)